services/src/handlers/datasets.rs

```diff
@@ -41,6 +58,31 @@ async fn list_datasets<C: Context>(
     Ok(warp::reply::json(&list))
 }
 
+pub(crate) fn get_dataset_handler<C: Context>(
```

Comment:
Maybe you can briefly add what this handler does.

Reply:
it gets a dataset by id... I'd like to do proper documentation of the API in a separate task